endpoints/api/manifest.py

```diff
@@ -120,7 +130,23 @@ def get(self, namespace_name, repository_name, manifestref):
         if manifest is None or manifest.internal_manifest_bytes.as_unicode() == "":
             raise NotFound()
 
-        return _manifest_dict(manifest)
+        manifest_dict = _manifest_dict(manifest)
+
+        if parsed_args["include_modelcard"]:
+            parsed = manifest.get_parsed_manifest()
+            layer_digest = str(parsed.filesystem_layers[-1].digest)
```

Comment:
How do we differentiate manifests that contain model cards to those that don't? We certainly don't want to try to read in the blob of images that don't contain a model card